samples/nrf9160/lwm2m_client/overlay-assist-cell.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_OBJ_SUPPORT=y
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_EVENTS=y
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_CELL=y
+CONFIG_LWM2M_VERSION_1_1=y
+CONFIG_BASE64=y
+CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
+CONFIG_LWM2M_RW_JSON_SUPPORT=y
```

Comment:
Same questions as with the previous overlay file.

Reply:
OLd JSON shuold remove